content/tutorials/interoperability/get-started-evm-sidechain.md

```diff
@@ -8,29 +8,30 @@ status: not_enabled
 ---
 # Get Started with the EVM Sidechain
 
-This getting started tutorial walks you through the steps to set up your account and transfer funds using the EVM sidechain bridge. 
+This getting started tutorial walks you through the steps to set up your account and submit a transaction using the EVM sidechain bridge. 
 
 ## 1. Create an Account Using an EVM Compatible Wallet
 
 In order to interact with the network, you need to create an account in the EVM sidechain. To create and manage this account you can use any EVM compatible wallet such as MetaMask.
 
 For instructions on how to install and create an account using MetaMask, then send and receive tokens, see [Connect MetaMask to XRP Ledger EVM Sidechain](connect-metamask-to-xrpl-evm-sidechain.html).
```

Comment:
Change from:
For instructions on how to install and create an account using MetaMask, then send and receive tokens,
To:
For instructions on how to install and create an account using MetaMask,